src/SFA.DAS.ApprenticeCommitments.Jobs.AcceptanceTests/Services/TestMessageBusBinding.cs

```diff
@@ -1,23 +1,62 @@
-﻿using NServiceBus.Transport;
+﻿using System;
+using System.IO;
+using NServiceBus.Transport;
 using System.Threading.Tasks;
 using TechTalk.SpecFlow;
 
 namespace SFA.DAS.ApprenticeCommitments.Jobs.AcceptanceTests.Services
 {
     [Binding]
-    //[Scope(Tag = "messageBus")]
     public class TestMessageBusBinding
     {
-        private readonly TestContext context;
+        private readonly TestContext _context;
 
-        public TestMessageBusBinding(TestContext context) => this.context = context;
+        public TestMessageBusBinding(TestContext context) => _context = context;
 
-        [BeforeScenario(Order = 1)]
+        [BeforeScenario(Order = 2)]
         public Task InitialiseMessageBus()
         {
-            context.TestMessageBus = new TestMessageBus();
-            context.Hooks.Add(new MessageBusHook<MessageContext>());
-            return context.TestMessageBus.Start(/*_context.TestDirectory*/);
+            _context.WorkingDirectory = GetWorkDirectory();
+            if (!_context.WorkingDirectory.Exists)
+            {
+                Directory.CreateDirectory(_context.WorkingDirectory.FullName);
```

Comment:
Maybe we can consolidate all the `CreateDirectory` calls to once place?  

The directory responsibility seems to be spread around across multiple places.  There may be a WorkingDirectory object hidden in there somewhere.